src/Routing/SubscriptionsRouteSubscriber.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Drupal\oe_subscriptions\Routing;
+
+use Drupal\Core\Routing\RouteSubscriberBase;
+use Symfony\Component\Routing\RouteCollection;
+
+/**
+ * Listens to the dynamic route events.
+ */
+class SubscriptionsRouteSubscriber extends RouteSubscriberBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function alterRoutes(RouteCollection $collection) {
+    $message_subscribe_routes = [
+      // Module: message_subscribe.
+      'message_subscribe.admin_settings',
```

Comment:
This should stay, only the user routes are duplicated for us.

Reply:
Removed